python/py_namespace.bzl

```diff
@@ -0,0 +1,141 @@
+"""Repository rule py_workspace(), augmenting @rules_python, for relocating
+py_library() and py_package() files underneath a given Python namespace.
+
+The stock @rules_python py_library() -> py_package() -> py_wheel() BUILD file
+workflow packages files at Python package paths set to the source paths of the
+files relative to the workspace root. This has a several problems. Firstly, it
+implies that files must be located underneath a source directory with the same
+name as the desired Python namespace package. ( py_wheel.strip_path_prefixes
+can remove path components, but cannot add them.) This is not always feasible
+or desirable.
+
+Secondly, this path naming is maddingly incompatible with the PYTHONPATH set by
```

Comment:
nit: "this path naming is maddingly incompatible" --> "this path naming is incompatible"

Reply:
Fixed in https://github.com/tensorflow/tflite-micro/pull/2151/commits/b6f04b387ad7466cb50d5463ceb6ce35ed713c7a.